libminifi/test/unit/OsUtilTests.cpp

```diff
@@ -39,6 +39,10 @@ TEST_CASE("Test userIdToUsername for well-known SIDs", "[OsUtils]") {
   CHECK_FALSE(minifi::utils::OsUtils::userIdToUsername("S-1-3-4").empty());
   CHECK_FALSE(minifi::utils::OsUtils::userIdToUsername("S-1-5-80-0").empty());
 }
+#else
+TEST_CASE("OsUtilTests") {
```

Comment:
We could also remove this from the file list in cmake instead.

Reply:
I didnt want to remove this from cmake because its reasonable that we would add non windows tests to this file (contrary to the others because those are inherently platform dependent). So instead I've added a test that checks if the machine arch is supported (this may become useful for more obscure platforms we might want to support in the future)

Reply:
Makes sense, thanks.